### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: c++/src/kj/compat/tls.c++ [hunk 1]
```diff
@@ -905,7 +905,7 @@ TlsCertificate::TlsCertificate(kj::ArrayPtr<const kj::ArrayPtr<const byte>> asn1
   for (auto i: kj::indices(asn1)) {
     auto p = asn1[i].begin();
 
-    // "_AUX" apparently refers to some auxilliary information that can be appended to the
+    // "_AUX" apparently refers to some auxillary information that can be appended to the
```
Comment (jclee) [inline c++/src/kj/compat/tls.c++:908]: I think `auxiliary` is actually the common spelling. (And again, below.)

(Nice job!)

Comment (rex4539) [reply c++/src/kj/compat/tls.c++]: Done :)

**Review by kentonv — COMMENTED** [review]
Wow, that's a lot of typos. How did you find all these?

Just one nitpick...

File: c++/src/capnp/rpc-test.c++ [hunk 2]
```diff
@@ -798,7 +798,7 @@ TEST(Rpc, TailCallCancelRace) {
   KJ_ASSERT(cancelCount == 1);
 }
 
-TEST(Rpc, Cancelation) {
+TEST(Rpc, Cancellation) {
```
[the remark this answers is not in the document]
Comment (rex4539) [reply c++/src/capnp/rpc-test.c++:801]: Actually, there were 3 more instances in this same file already spelled `Cancellation` and I only changed this for consistency :)

[the remark this answers is not in the document]
Comment (rex4539) [reply c++/src/capnp/rpc-test.c++:801]: And guess who committed the other 3 instances? (wink, wink)

[the remark this answers is not in the document]
Comment (jclee) [reply c++/src/capnp/rpc-test.c++:801]: Actually, "cancellation" seems to be the preferred spelling in both British and American spelling.  It's "canceled" vs. "cancelled" and "canceling" vs. "cancelling" where the two actually differ.  https://www.grammarly.com/blog/canceled-vs-cancelled/

**Pull request merged**